src/js/HRModules/addForm.js

```diff
@@ -7,10 +7,48 @@ const {
     createNumberQuestion
 } = require('../HRModules/newForm');
 const { Validate } = require('../validator');
+const { getOneSynonym, getSynonyms, getLanguages } = require('../common/yandex');
+
+let focused;
+
 
 const AddForm = {
     questions: [],
 
+    getText(elem) {
+        if(elem.tagName === "INPUT" && elem.type === "text") {
+            return elem.value.substring(elem.selectionStart,
+                    elem.selectionEnd);
+        }
+        return null;
+    },
+
+    findSynonym() {
+        Promise.resolve(getOneSynonym(focused))
+            .then(str => {
+                if (str.length === 0) {
+                    Dialogs.alert('Bład', 'Nie ma takiego słowa w bazie');
+                    return;
+                }
+                const firstSynonym = str[0].tr[0].text;
+                Promise.resolve(getSynonyms(firstSynonym))
+                    .then(tab => {
+                        let synonyms = '';
+                        for (speech_parts of tab) {
+                            if (speech_parts.tr[0].syn) {
+                                for (obj of speech_parts.tr[0].syn) {
+                                    synonyms += obj.text + ', '
```

Comment:
Tym sposobem zawsze będzie ',' na końcku

Reply:
wiem, ale uznalam ze to nie ma znaczenia